dadi/lib/controller/index.js

```diff
@@ -185,6 +186,14 @@ const Controller = function (router) {
   router.post('/api/routes', function (req, res) {
     return RouteController.post(req, res)
   })
+
+  router.use('/_dadi/domains/:domain?', function (req, res, next) {
```

Comment:
Should we put this behind a configuration option (like `dadiNetwork.enableConfigurationAPI`) so that it's disabled by default?